apps/selvbetjening-ui/package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "selvbetjening-ui-client",
+  "version": "0.1.0",
+  "private": true,
+  "homepage": "/selvbetjening",
+  "type": "module",
+  "dependencies": {
+    "@navikt/aksel-icons": "^6.11.0",
```

Comment:
Hvis man kan skrive seg bort fra denne, så hadde det vært nice 😄

Reply:
Har vi noen bedre alternativer?

Reply:
nvm, blandet den med en gammel pakke vi brukte for ikoner..